### DISCUSSION TITLE
슬라이드 공지는 따로 DB 컬럼을 두어 개발할까요??

### DISCUSSION THREAD

**Discussion by wlswo** [MoCo-v2/moco-v2-backend#5] [Q&A] [answered]
어떤 방식이 괜찮으신가요?

**Comment by HJ97Kim** [comment, marked as answer]
따로 테이블이나 컬럼을 두는 방식이 좋아보여요.
공지특성상 자주 바뀌는 것이 아니라면 하드코딩도 방법이 될 순 있겠지만 자주 바뀔 빈도수를 생각하면 새로 컬럼이나 테이블을 두는게 적합해 보입니다.
전자의 경우 새롭게 배포를 진행해야 하는 번거로움이 있을수도 있을 것 같아요
